app/components/bali/calendar/component.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="calendar-component">
+  <div class="box <%= period == :month ? 'month-view' : 'week-view' %>">
+    <%= header %>
+
+    <div class="table-container">
+      <table class="table is-fullwidth">
```

Comment:
This `table` should be using `Bali::Table::Component`

Reply:
The table component is not ready for this use, in the calendar component the `tr` and `td` tags need to use special classes and since the table component doesn't accept dynamic classes in the `tr` and `td` tags, the table component is useless here.